src/components/study-pane.js

```diff
@@ -430,26 +430,72 @@ const StudyPane = (props) => {
         [studyName, userId]
     );
 
+    const updateEquipment = useCallback(
+        (typeEquipment, equipmentId) => {
+            if (!typeEquipment || !equipmentId) {
+                return;
+            }
+            let fetchCb, equipments;
+            switch (typeEquipment) {
+                case 'GENERATOR':
+                    fetchCb = fetchGenerators(studyName, userId);
+                    equipments = network.generators;
+                    break;
+                case 'TWO_WINDING_TRANSFORMER':
```

Comment:
WINDINGS (to be consistent with changes requested in gridsuite/network-modification-server#10)